File: jaea-kim/Week_02/BOJ_10814_나이순정렬.java
package Week_02;

import java.util.*;

public class BOJ_10814_나이순정렬 {
    static class Member {
        int age;
        String name;

        public Member(int age, String name) {
            this.age = age;
            this.name = name;
        }

        @Override
        public String toString() {
            return age + " " + name;
        }
    }
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int num = sc.nextInt();
        sc.nextLine();

        Member[] members = new Member[num];

        for (int i = 0; i < num; i++) {
            String[] input = sc.nextLine().split(" ");
            members[i] = new Member(Integer.parseInt(input[0]), input[1]);
        }

        Arrays.sort(members, new Comparator<Member>() {
            @Override
            public int compare(Member o1, Member o2) {
                return Integer.compare(o1.age, o2.age);
            }
        });

        for (Member member : members){
            System.out.println(member.toString());
        }

    }
}

File: JeonHyoChang/Season1/Week_02/Python/BOJ_10814_나이순정렬.py
import sys

N = int(sys.stdin.readline())
user_list = list()

for _ in range(N):
    age, name = map(str, sys.stdin.readline().split())
    age = int(age)
    user_list.append((age, name))

user_list.sort(key=lambda x: x[0])

for e in user_list:
    print(e[0], e[1])
